docs/usage/feeds.md

```diff
@@ -1,48 +1,73 @@
 # Native feeds
 
+OpenCTI provides versatile mechanisms for sharing data through its built-in feeds, including Live Stream, TAXII collection, and CSV feeds.
+
+## Feed configuration
+
+Feeds are configured in the "Data > Data sharing" window. Configuration for all feed types is uniform and relies on the following parameters:
+
+- Filter setup: The feed can have specific filters to publish only a subset of the platform overall knowledge. Any data that meets the criteria established by the user's feed filters will be shared (e.g. specific types of entities, labels, marking definitions, etc.).
+- Access control: A feed can be either public, i.e. accessible without authentication, or restricted. By default, it's accessible to any user with the "Access data sharing" capability, but it's possible to increase restrictions by limiting access to a specific user, group, or organization.
+
+![Feed access restriction](assets/feed-access-restriction.png)
+
+By carefully configuring filters and access controls, you can tailor the behavior of Live Stream, TAXII collection, and CSV feeds to align with your specific data-sharing needs.
+
+
 ## Live streams
 
 ### Introduction
 
-The best way to consume OpenCTI data, whether it is through a [stream connector](../deployment/connectors.md#stream-section) or within another OpenCTI instance, is to use the live streams. Live streams are like TAXII collection (ie. serving STIX 2.1 bundles) but *under steroids*. This means that live streams are supporting:
+Live Streams, an exclusive OpenCTI feature, increase the capacity for real-time data sharing by serving STIX 2.1 bundles as TAXII collections with advanced capabilities. What distinguishes them is their dynamic nature, which includes the creation, updating, and deletion of data. Unlike TAXII, Live Streams comprehensively resolve relationships and dependencies, ensuring a more nuanced and interconnected exchange of information. This is particularly beneficial in scenarios where sharing involves entities with complex relationships, providing a richer context for the shared data. When sharing data between two OpenCTI platforms, Live Streams are the preferred mechanism.
 
-* create, update and delete events depending on the filters ;
-* caching already created entities in the last 5 minutes ;
-* resolving relationships and dependencies even out of the filters ;
-* they can be public (without authentication).
+In scenarios involving data sharing between two OpenCTI platforms, Live Streams emerge as the preferred mechanism. Consuming OpenCTI data, whether through a [stream connector](../deployment/connectors.md#stream-section) or within another OpenCTI instance, is most efficiently achieved through Live Streams. These streams operate like TAXII collections but are notably enhanced, supporting:
```

Comment:
it feels like the same thing said three times:
- When sharing data between two OpenCTI platforms, Live Streams are the preferred mechanism.
- In scenarios involving data sharing between two OpenCTI platforms, Live Streams emerge as the preferred mechanism.
- Consuming OpenCTI data, whether through a [stream connector](https://docs.opencti.io/latest/deployment/connectors/#stream-section) or within another OpenCTI instance, is most efficiently achieved through Live Streams.

Reply:
Indeed!